server/src/routes/api/v1/dogParksRouter.js

```diff
@@ -44,4 +43,25 @@ dogParksRouter.get("/:id", async (req, res) => {
     }
 })
 
+dogParksRouter.post("/:id", async (req, res) => {  
+    try {
+        const { body } = req
+        const userId = req.user.id
+        body.review.userId = userId
+        const { id } = req.params
+        body.review.dogParkId = id
+        const cleanedInput = cleanUserInput(body.review)
```

Comment:
we should use `cleanUserInput` with just the submitted data to remove empty values. We can then attach the other properties userId and park id to the `cleanedInput` so that we do not modify the incoming `req.body`